docs/glossary.md

```diff
@@ -0,0 +1,97 @@
+# Glossary
+
+Definition of commonly used terms in the context of this Romcal project. The terms are listed in alphabetical order.
+
+## Celebration
```

Comment:
```suggestion
## Liturgical Celebration
```

Reply:
Actually, just like with "Rank," I'm questioning the relevance of prefixing with the word "Liturgical." Since in the context of this project, a celebration is inherently liturgical. And when we eventually have objects of type `Celebration` in the codebase, it's also good to keep names from being too long.

Reply:
I agree, I read it a few times, thought about it.

Reducing verbosity spread on names becomes a real problem when you've got as many types as we do

Reply:
> Reducing verbosity spread on names becomes a real problem when you've got as many types as we do

I personally don’t think the verbosity and length are an issue for this glossary of ours. It should contain as many terms as we / the Church use (-s). If we (can) use multiple terms for the same thing, we should add a glossary item for each, all (but one) linking to the main one (e.g. `Feria` should refer to `Weekday` or `GIRM` to `General Instruction on the Roman Missal`).

Reply:
Then why not link "Liturgical Celebration" to "Celebration" instead of extending the main definition, requiring a reordering as well